### PR TITLE
Add a new file for testing and learning about git workflow

### PR DESCRIPTION
**Pull request by nguyenla**
An, could you please help me review this change and pull it into your master branch?

### CODE REVIEW

**Review by alfienguyen22 — CHANGES_REQUESTED** [review]
Please add Javascript for Signup process and alternative for multiple line breaks.

File: index_2.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                <div class="container">
+                    <span class="psw"><a href="#"> Quên Mật Khẩu? </a></span><br><br>
+                    <a href="signup.html">Đăng Ký Ngay!</a>
+                </div>
+            </form>
+        </div>
+      <br>
+      <br>
+      <br>
+      <br>
```
Comment (alfienguyen22) [inline index_2.html:54]: Please find an alternative line break option.

Comment (nguyenla) [reply index_2.html:54]: done

**Comment by nguyenla** [comment]
I will create the Sign up page in a separate pull request.

**Review by alfienguyen22 — APPROVED** [review]
Looks good to me!

**Pull request merged**